core/templates/dev/head/components/summary_tile/canonical_story_summary_tile_directive.html

```diff
@@ -0,0 +1,44 @@
+<a ng-href="<[getStoryLink()]>">
+  <div class="oppia-story-image">
+    <img src="https://image.ibb.co/iUeEUJ/Screenshot_from_2018_07_21_11_07_04.png" alt="">
+  </div>
+  <div class="oppia-story-details">
+    <div class="oppia-story-title">
+      <[getStoryTitle()]>
+    </div>
+    <div class="oppia-story-description">
+      <[getStoryDescription()]>
+    </div>
+  </div>
+</a>
+<style>
+  canonical-story-summary-tile a {
+    padding: 20px;
+  }
+  canonical-story-summary-tile .oppia-story-image {
+    padding: 0 34px 0 34px;
+  }
+  canonical-story-summary-tile .oppia-story-details {
+    padding: 10px;
+  }
+  canonical-story-summary-tile .oppia-story-title {
+    display: block;
+    color: #4a4a4a;
+    text-align: center;
+    font-size: 1.4em;
+    font-weight: bold;
+    padding-bottom: 20px;
+  }
+  canonical-story-summary-tile .oppia-story-description {
+    display: block;
+    color: #4a4a4a;
+    text-align: center;
+    font-size: 1.1em;
+    position: absolute;
+    bottom: 10px;
+    padding-right: 10px;
+  }
+  a:hover {
```

Comment:
Have this also be enclosed in the directive tag.

Reply:
Okay